lib/commands/instrument.js

```diff
@@ -73,6 +78,7 @@ exports.handler = function (argv) {
     produceSourceMap: argv.produceSourceMap,
     extension: argv.extension,
     require: argv.require,
+    cwd: argv.cwd,
```

Comment:
This we do need to properly configure the `test-exclude` instance used to determine which files should be instrumented.  This makes a difference if `package.json` is at `..` (you are in a subdirectory of the project).